flang/include/flang/Common/uint128.h

```diff
@@ -255,7 +266,7 @@ template <bool IS_SIGNED = false> class Int128 {
     }
   }
   static constexpr std::uint64_t topBit{std::uint64_t{1} << 63};
-  std::uint64_t low_{0}, high_{0};
+  std::uint64_t SWAP(low_, 0, high_, 0);
```

Comment:
This line is the declaration that should be wrapped with `#if FLANG_LITTLE_ENDIAN` &c.